worlds/spelunker/local_data.py

```diff
@@ -0,0 +1,189 @@
+local_locations = {

```

Comment:
```suggestion
from typing import Dict, List

local_locations: Dict[str, int] = {

```
You have typing everywhere else, should probably have it here too.

Also I'm not sure if these count as ints or not, so you should verify if int is correct

Reply:
Fixed